members/serializers.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from members.models import Member
+from members.permissions import has_permission
+from core.serializers import SemesterSerializer
+
+
+class MemberSerializer(serializers.ModelSerializer):
+    semester = SemesterSerializer(read_only=True)
+
+
+
+    class Meta:
+        model = Member
+        fields = ('id', 'name', 'email', 'date_joined', 'semester', 'lifetime', 'honorary')
+
+
+class AddMemberSerializer(serializers.Serializer):
+    '''
+    class Meta:
+        model = Member
+        field = ('id', 'name', 'email', 'lifetime', 'honorary', 'user')
+    '''
+    name = serializers.CharField(max_length=50, help_text='name of user, max 50 letters')
```

Comment:
Ja, det er forsåvidt en god idé. Legger det inn som et optional felt i apiet for nå, så kan man gjøre de forandringene du nevner senere.